jenkins/scripts/bare_metal_lab/tasks/pod_scaling/pod-scaling.yaml

```diff
@@ -0,0 +1,206 @@
+      wait_timeout: 1200
+      kubeconfig: /home/metal3/.kube/config
+
+  - name: Scale up NGINX deployment to 200
+    kubernetes.core.k8s_scale:
+      api_version: v1
+      kind: Deployment
+      name: nginx-deployment
+      namespace: pod-scaling-test
+      replicas: 200
+      wait_timeout: 1200
+      kubeconfig: /home/metal3/.kube/config
+
+  - name: Scale up NGINX deployment to 250
+    kubernetes.core.k8s_scale:
+      api_version: v1
+      kind: Deployment
+      name: nginx-deployment
+      namespace: pod-scaling-test
+      replicas: 250
+      wait_timeout: 1200
+      kubeconfig: /home/metal3/.kube/config
+
+  - name: Scale up NGINX deployment to 300
+    kubernetes.core.k8s_scale:
+      api_version: v1
+      kind: Deployment
+      name: nginx-deployment
+      namespace: pod-scaling-test
+      replicas: 300
+      wait_timeout: 1200
+      kubeconfig: /home/metal3/.kube/config
+
+  - name: Scale up NGINX deployment to 350
+    kubernetes.core.k8s_scale:
+      api_version: v1
+      kind: Deployment
+      name: nginx-deployment
+      namespace: pod-scaling-test
+      replicas: 350
+      wait_timeout: 1200
+      kubeconfig: /home/metal3/.kube/config
+
+  - name: Scale up NGINX deployment to 400
+    kubernetes.core.k8s_scale:
+      api_version: v1
+      kind: Deployment
+      name: nginx-deployment
+      namespace: pod-scaling-test
+      replicas: 400
+      wait_timeout: 1200
+      kubeconfig: /home/metal3/.kube/config
+
+  - name: Scale up NGINX deployment to 450
+    kubernetes.core.k8s_scale:
+      api_version: v1
+      kind: Deployment
+      name: nginx-deployment
+      namespace: pod-scaling-test
+      replicas: 450
+      wait_timeout: 1200
+      kubeconfig: /home/metal3/.kube/config
+
+  - name: Scale up NGINX deployment to 500
+    kubernetes.core.k8s_scale:
+      api_version: v1
+      kind: Deployment
+      name: nginx-deployment
+      namespace: pod-scaling-test
+      replicas: 500
+      wait_timeout: 1200
+      kubeconfig: /home/metal3/.kube/config
+
+  - name: Scale up NGINX deployment to 550
+    kubernetes.core.k8s_scale:
+      api_version: v1
+      kind: Deployment
+      name: nginx-deployment
+      namespace: pod-scaling-test
+      replicas: 550
+      wait_timeout: 1200
+      kubeconfig: /home/metal3/.kube/config
+
+  - name: Scale up NGINX deployment to 600
+    kubernetes.core.k8s_scale:
+      api_version: v1
+      kind: Deployment
+      name: nginx-deployment
+      namespace: pod-scaling-test
+      replicas: 600
+      wait_timeout: 1200
+      kubeconfig: /home/metal3/.kube/config
+
+  - name: Scale up NGINX deployment to 650
+    kubernetes.core.k8s_scale:
+      api_version: v1
+      kind: Deployment
+      name: nginx-deployment
+      namespace: pod-scaling-test
+      replicas: 650
+      wait_timeout: 1200
+      kubeconfig: /home/metal3/.kube/config
+
+  - name: Pause for 5 minutes to run 650 pods simultaneously
+    ansible.builtin.pause:
+      minutes: 5
```

Comment:
I think we need to do more than just pausing here. We should check that the Kubernetes API is stable, that the nodes are healthy and that the pods are running.

Reply:
I get your point. But this is going to mess up metal3 dev env repo again. My goal is to not touch other repos. We already have to many things put in metal3 dev env.